lib/ip_wrangler/exec.rb

```diff
@@ -0,0 +1,17 @@
+module IpWrangler
+  module Exec
+    $iptables_bin_path = '/usr/bin/sudo /sbin/iptables'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.